test/java/seedu/addressbook/storage/StorageFileTest.java

```diff
@@ -0,0 +1,109 @@
+package seedu.addressbook.storage;
+
+import static org.junit.Assert.assertEquals;
+import java.nio.file.Paths;
+import java.util.Collections;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.storage.StorageFile.StorageOperationException;
+import static seedu.addressbook.util.TestUtil.assertTextFilesEqual;
+
+public class StorageFileTest {
+    private static final String TEST_DATA_FOLDER = "test/data/StorageFileTest";
+    private static final String TEST_TEMP = "temp.txt";
```

Comment:
I requested it because it was duplicated in two places and was not DRY. Changing one required changing the other.  If we keep the constant, the name should be improved. But I'm ok to inline it too.